src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx

```diff
@@ -34,33 +34,39 @@ interface ArtQuizArtworksProps {
 }
 
 export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
+  const { submitMutation: submitDislike } = useDislikeArtwork()
+  const { submitMutation: submitSave } = useSaveArtwork()
+  const { submitMutation: submitUpdate } = useUpdateQuiz()
   const { router } = useRouter()
 
-  const edges = me.quiz.quizArtworkConnection!.edges!
-  const startingIndex = edges.findIndex(edge => {
+  // clone to make the array writable for sorting
+  const edges = me.quiz.quizArtworkConnection?.edges
+    ? [...me.quiz.quizArtworkConnection?.edges]
+    : []
+
+  // sort the artworks by position, ascending
+  const sortedEdges = useMemo(() => {
+    return edges.sort((a, b) => {
+      return a && b ? a.position - b.position : 0
+    })
+  }, [edges])
```

Comment:
Totally fine, but should we just support a sort in the query? `sort: POSITION_ASC`

Reply:
This is why they pay you the big bucks. 🧠

I'll make a ticket for the changes in MP.